### ISSUE TITLE
Scanning the QR code sometimes fails to scan the smaller QR code

### ISSUE THREAD

**Issue by mulieli** [pdliuw/ai_barcode#22]
Scanning the QR code sometimes fails to scan the smaller QR code

**Comment by mulieli** [comment]
And The color QR code is sometimes not scanned

**Comment by Aerofluxx** [comment]
Yes! Having the same issue, scanning works with ie. the "camera app" of the smartphone, but not with ai_barcode, with smaller qr codes.
